espnet2/asr/frontend/s3prl.py

```diff
@@ -82,12 +87,26 @@ def _get_upstream(self, frontend_conf):
         ]:
             s3prl_upstream.model.encoder.layerdrop = 0.0
 
+            # check if adapter is added
+            if (
+                s3prl_upstream.model.__class__.__name__ == "Wav2Vec2Model"
+                and self.args.add_adapters
+            ):
+                s3prl_upstream = add_adapters_wav2vec2(
+                    s3prl_upstream, adapter_down_dim=192, adapt_layers=[0, 1, 2]
```

Comment:
I suggest to make the adapter_down_dim, adapt_layers to be configurable.

Reply:
This a very good suggestion , will incorporate it soon

Reply:
Both` adapter_down_dim` and `adapt_layers` are no configurable from the config.yaml file

Reply:
@jwrh please test this by , pulling my master into your branch and run it till training starts to ensure the pipeline works. Also try changing the `adapter_down_dim' and `adapt_layers` to a different check if the model created is accurate

Reply:
@simpleoier we plan to add support to HUBERT and wavLM in the next PR. 
This PR we would like to keep to wav2vec2 and configurable adapter input.